pkg/kubelet/kuberuntime/helpers_windows.go

```diff
@@ -30,6 +30,7 @@ const (
 	milliCPUToCPU    = 1000
 )
 
+// TODO: remove - may be dead code
```

Comment:
This is dead code now that `kuberuntime_container_windows.go` doesnt call it

Reply:
removed